demo.ipynb

```diff
@@ -2,7 +2,29 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```

Comment:
Should the notebook be moved to the `samples/` directory and references updated or are we in general keeping all sample notebooks in the root directory?

Reply:
The notebook uses some modules from the core model code and if I put the notebook in samples folder, I wasn't able to import it.